README.md

```diff
@@ -58,16 +58,27 @@ Docker Note: If you make any changes to your `.env` file, you will need to resta
 
 ### Discord Bot: Python Script 🏃‍♀️
 1. Install Python 3.10 or higher
-2. Install the required packages:
+2. Setup a python virtual environment:
+#### Linux & Mac
+```bash
+python -m venv autorsa
+source ./autorsa/Scripts/activate
```

Comment:
its should be ./autorsa/bin/activate

Reply:
![image](https://github.com/user-attachments/assets/1c809f5b-682b-4256-b5b9-68c2b8c6e678)

It must be sourced as seen when looking at the script's code.

Reply:
i meant that the path is bin not scripts